cmd/mongocli/mongocli.go

```diff
@@ -38,6 +40,59 @@ func Execute(ctx context.Context) {
 	}
 }
 
+func updateMongoCLIConfigPath() {
+	mongoCLIConfigHome, err := config.MongoCLIConfigHome()
+	if err != nil {
+		return
+	}
+
+	mongoCLIConfigPath := fmt.Sprintf("%s/%s", mongoCLIConfigHome, "config.toml")
```

Comment:
I'm just realising this, but, does this work for windows or do we need to use `filepath.Join`?

Reply:
I would assume it works since `MongoCLIConfigHome` has been using this already? How can I confirm if this works?

Reply:
1. spawn a windows host on evergreen
2. run the task package mongocli msi on evergreen
3. access your windows host with Microsoft Remote Desktop
4. install the msi created at step 2, on your windows machine

Reply:
Thanks! On it

Reply:
Ok, tested and we're creating the right folder structure if that's the concern here:

```
PS C:\Users\Administrator\AppData\Roaming> ls


    Directory: C:\Users\Administrator\AppData\Roaming


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----        1/31/2022   5:40 PM                ActiveState
d-----         3/7/2022   4:06 PM                Adobe
d---s-         3/7/2022   4:06 PM                Microsoft
d-----         3/7/2022   4:21 PM                mongocli
d-----        1/31/2022   5:57 PM                npm
d-----        1/31/2022   5:31 PM                NuGet
d-----        1/31/2022   5:51 PM                Visual Studio Setup

```
```
PS C:\Users\Administrator\AppData\Roaming> cd mongocli
PS C:\Users\Administrator\AppData\Roaming\mongocli> ls


    Directory: C:\Users\Administrator\AppData\Roaming\mongocli


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----         3/7/2022   4:21 PM             45 config.toml

```
```
PS C:\Users\Administrator\AppData\Roaming\mongocli> cat config.toml

[default]
  output = ""
  service = "cloud"
```

Reply:
that looks like powershell, I may give this a try in `cmd` but let's not block the PR on this